### DISCUSSION TITLE
How do I override a single string in _data/ui-text.yml

### DISCUSSION THREAD

**Discussion by GregWoods** [mmistakes/minimal-mistakes#2456] [Q&A] [answered]
I have searched through the docs, and I have found references to setting strings in _data/ui-text.yml, but I can't get it working.
I am using a remote theme, with a some small changes to _layout\archive and _includes\archive-single.html. 
I like the way I can copy over a single file in the theme and make my changes, and it will prefer my file over the remote one. But the same logic doesn't seem to work with ui-text.yml
I have tried modifying just one string, and I've tried copying the entire file and making my alteration. Neither is working for me.

Any help appreciated

Jekyll 3.8.5
minimal-mistakes 4.19.0

**Comment by mmistakes** [comment, marked as answer]
Same logic works for `_data` files too. Any text you override in that file will be used by the theme.
Can you provide a link to a public repo? Common issues is if you edited that YAML file and it's not valid anymore (incorrect indents or a bad string). 

Also did you make your text edits under the correct `locale` that corresponds to what you have set in `_config.yml`

**Comment by GregWoods** [comment]
I had multiple issues.
* An indenting error after the copy/paste
* I had assigned to the default `en` locale instead of `en-GB`, which would have been Ok, had I left in the :
```
en-GB:
  <<: *DEFAULT_EN
```
at the end.

Thanks pointing out the obvious gotchas. It was a great help just knowing I was on the right lines!
